app/controllers/walks_controller.rb

```diff
@@ -14,7 +14,7 @@ def create
     @walk = Walk.new(walks_params)
     @walk.requester_id = current_user.id
 
-    if @walk.save
+    if @walk.save  
```

Comment:
I think I added trailing whitespace. We shouldn't have that. If you see this in my code, comment on it so I can fix it please.